### PR TITLE
chore: improve insufficient gas ux flow

### LINKED ISSUE #889: DEX - Not have the funds to pay for the gas fee's

### PR DESCRIPTION
**Pull request by ThomasRalee**
### This PR:
- resolves #889 

### Screenshot
image

### CODE REVIEW

**Comment by ThomasRalee** [comment]
### Tested the UX flow on Keplr:
https://www.loom.com/share/7d5224de863749ad875ef1a21d35d359

**Comment by bangjelkoski** [comment]
When can we merge? @ThomasRalee @Tofel

File: components/partials/fee-discounts/fees/index.vue [hunk 1]
```diff
@@ -101,8 +101,7 @@ export default Vue.extend({
           makerDiscountRate: '0',
           takerDiscountRate: '0',
           stakedAmount: '0',
-          feePaidAmount: '0',
-          volume: '0'
+          feePaidAmount: '0'
```
Comment (bangjelkoski) [inline components/partials/fee-discounts/fees/index.vue:104]: Why remove `volume`?

**Review by bangjelkoski — CHANGES_REQUESTED** [review]

File: components/partials/fee-discounts/fees/index.vue (hunk 1 above)
Comment (ThomasRalee) [reply components/partials/fee-discounts/fees/index.vue]: the build was complaining, just added it back after updating the `sdk-ui` and `sdk-ui-ts` package 👍

**Pull request merged**